tests/simple_example.rs

```diff
@@ -180,3 +180,36 @@ where
 
     Ok(vec![root_key.key_id().clone()])
 }
+
+// FIXME(232): This is a helper function that, if `consistent_snapshot` is true, it prefixes target
+// paths with their file hash. This is unfortunately necessary because we don't have a high level
+// interface for creating TUF metadata.
+async fn store_target(
+    remote: &EphemeralRepository<Json>,
+    target_file: &[u8],
+    target_path: &TargetPath,
+    consistent_snapshot: bool,
+) -> Result<()> {
+    if consistent_snapshot {
+        let target_path = hash_prefix_target_path(target_file, target_path)?;
+        remote.store_target(target_file, &target_path).await
+    } else {
+        remote.store_target(target_file, target_path).await
+    }
+}
+
+fn hash_prefix_target_path(target_file: &[u8], target: &TargetPath) -> Result<TargetPath> {
+    let target_description = TargetDescription::from_reader(target_file, &[HashAlgorithm::Sha256])?;
+    let (_, value) = crypto::hash_preference(target_description.hashes())?;
+
+    let mut components = target.components();
+    let file_name = match components.pop() {
+        Some(file_name) => file_name,
+        None => {
+            return Err(Error::IllegalArgument("Path cannot be empty".into()));
+        }
+    };
+    components.push(format!("{}.{}", value, file_name));
+
+    TargetPath::new(components.join("/"))
+}
```

Comment:
Rather than do all this, could you just hardcode the path, since it's only used once? That'd probably make the test easier to understand, too.

Reply:
I cleaned this up, like it better?